### PR TITLE
docs: Updates license to hippocratic license.

### LINKED ISSUE #53: License Discussion
What are your opinions on our license? Is it odd that someone can sell our software if they wanted to? Also, if there's a possibility that someone could use our software for nefarious purposes should we switch to the hippocratic license?

### PR DESCRIPTION
**Pull request by christopheralex**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,22 +1,213 @@
-MIT License
+[airpyllution] Copyright (2022) (Christopher Alexander, Daniel King, Mel Liow)(“Licensor”)
```
Comment (danfke) [inline LICENSE]: I think it should be `airpyllution Copyright 2022 Christopher Alexander, Daniel King, Mel Liow` Not sure whether to include (“Licensor”) or not

**Comment by danfke** [comment]
The README's reference to the license has to be updated

File: LICENSE (hunk 1 above)
Comment (christopheralex) [reply LICENSE]: Done

**Comment by christopheralex** [comment]
> The README's reference to the license has to be updated

Done

File: README.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ get_pollution_forecast(49.28, 123.12, api_key)
 Interested in contributing? Check out the contributing guidelines. Please note that this project is released with a Code of Conduct. By contributing to this project, you agree to abide by its terms.
 ## License
 
-`airpyllution` was created by Christopher Alexander, Daniel King, Mel Liow. It is licensed under the terms of the MIT license.
+`airpyllution` was created by Christopher Alexander, Daniel King, Mel Liow. It is licensed under the terms of [Hippocratic License 3.0](https://github.com/UBC-MDS/airpyllution/blob/main/LICENSE).
```
Comment (danfke) [inline README.md]: minor detail: should be `the [ Hippocratic License 3.0]`

**Review by danfke — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (christopheralex) [reply README.md]: Done.

**Pull request merged**